### PR TITLE
Fix/content layout design review

### PR DESCRIPTION
**Pull request by alexanderleegs**
Resolves https://linear.app/ogp/issue/ISOM-962/content-layout-design-review.

### CODE REVIEW

**Comment by dcshzj** [comment]
!run chromatic

File: src/types/page.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export interface ContentPageProps extends BasePageProps {
     ContentPageHeaderProps,
     "summary" | "buttonLabel" | "buttonUrl"
   >
+  lastUpdated: string
```
[the remark this answers is not in the document]
Comment (alexanderleegs) [reply src/types/page.ts]: Changed in https://github.com/isomerpages/isomer-components/pull/88/commits/8fa5fa73a1faefe3adc6e74b8c9963ed3f5a0fb5!

**Comment by alexanderleegs** [comment]
tagging @sehyunidaaa !

File: src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx [hunk 2]
```diff
@@ -7,21 +7,29 @@ import { Paragraph } from "../../../typography/Paragraph"
...
           <h1 className="text-[2.75rem] leading-tight lg:text-[3.75rem] font-semibold text-content-strong">
             {title}
           </h1>
+          <div className="pt-6 lg:pb-2">{`Last updated ${new Date(
+            lastUpdated,
+          ).toLocaleDateString(undefined, {
+            day: "2-digit",
+            month: "short",
+            year: "numeric",
+          })}`}</div>
```
Comment (dcshzj) [inline src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx:32]: nit: I think there might be some value in us standardising the date format throughout the entire site. Can I trouble you to extract out this as a util function and update the footer + article layout as well?

File: src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx [hunk 3]
```diff
@@ -7,21 +7,29 @@ import { Paragraph } from "../../../typography/Paragraph"
...
   summary,
+  lastUpdated,
   breadcrumb,
   buttonLabel,
   buttonUrl,
   LinkComponent,
 }: ContentPageHeaderProps) => {
   return (
-    <div className="bg-site-primary-100 px-4 py-8 lg:px-5 lg:py-16">
+    <div className="bg-site-primary-100 px-4 py-8 md:py-10 lg:px-5 lg:py-16">
```
Comment (dcshzj) [inline src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx:17]: issue: I think it should be `px` that should be changed?

File: src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx (hunk 2 above)
Comment (alexanderleegs) [reply src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx]: Refactored in https://github.com/isomerpages/isomer-components/pull/88/commits/790058b20a8b97ef7f0b147e159f0d125d65b5ea!

File: src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx (hunk 3 above)
Comment (alexanderleegs) [reply src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx]: Confirmed with Se Hyun - have fixed it in https://github.com/isomerpages/isomer-components/pull/88/commits/c4a7878aec9f2a41c0a3522627dbd824d4d446b8!

File: src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx [hunk 4]
```diff
@@ -3,25 +3,28 @@ import Button from "../../complex/Button"
 import Breadcrumb from "../Breadcrumb"
 import BaseParagraph from "../BaseParagraph"
 import { Paragraph } from "../../../typography/Paragraph"
+import { getFormattedDate } from "~/utils/getFormattedDate"
```
Comment (dcshzj) [inline src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx:6]: omega nit: can we import only from `~/utils`, so we export this inside `~/utils/index.ts`

**Review by dcshzj — APPROVED** [review]

File: src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx (hunk 4 above)
Comment (alexanderleegs) [reply src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx]: Fixed in https://github.com/isomerpages/isomer-components/pull/88/commits/78358336580097e5134b964a0f4942822891c994!

**Pull request merged**